regimes/it/charges.go

```diff
@@ -0,0 +1,26 @@
+package it
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/tax"
+)
+
+// List of charge types specific to the italian regime.
+const (
+	ChargeKeyStampDuty cbc.Key = "stamp-duty"
+)
+
+var chargeKeys = []*tax.KeyDefinition{
+	{
+		Key: ChargeKeyStampDuty,
+		Name: i18n.String{
+			i18n.EN: "Stamp Stamp",
+			i18n.IT: "Bollo",
```

Comment:
Should the name be "tassa di bollo"?

```suggestion
			i18n.IT: "Tassa di Bollo",
```